code-challenges.js

```diff
@@ -31,25 +59,74 @@ const temperature2 = 350
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilPoint = (theTemp) => {
+
+    if(theTemp > 212){
+        return `${theTemp} is above boiling point`
+    } else if (theTemp < 212){
+        return `${theTemp} is below boiling point`
+    } else{
+        return `${theTemp} is at boiling point`
+    }
+}
```

Comment:
Great! Nice work! Quick note, your pseudo code says you are concatenating the original number with the rest of the text, but you are actually using string interpolation. Make sure your code reflects your pseudo code!